client-react/src/features/todos/models/todoInputs.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface TodoInputs {
```

Comment:
Keep naming convension the same accross the project. Use kebab-case

Reply:
fixed